source/jormungandr/jormungandr/interfaces/v1/Places.py

```diff
@@ -86,19 +164,107 @@ def __init__(self, *args, **kwargs):
                                 default=False)
 
     @use_old_disruptions_if_needed()
-    @marshal_with(places)
     def get(self, region=None, lon=None, lat=None):
-        self.region = i_manager.get_region(region, lon, lat)
         args = self.parsers["get"].parse_args()
-        g.use_old_disruptions = args['_use_old_disruptions']
         self._register_interpreted_parameters(args)
-
+        g.use_old_disruptions = args['_use_old_disruptions']
         if len(args['q']) == 0:
             abort(400, message="Search word absent")
-        response = i_manager.dispatch(args, "places",
-                                      instance_name=self.region)
+
+        if not any([region, lon, lat]):
+            return self.get_ww(args)
+        else:
+            return self.get_region(args, region, lon, lat)
+
+    @marshal_with(places)
+    def get_region(self, args, region=None, lon=None, lat=None):
```

Comment:
I don't think this method is correctly named (and there is another get_region method which get the region :p)